src/main/java/com/conveyal/datatools/manager/auth/Auth0Users.java

```diff
@@ -45,16 +45,22 @@
     private static final ObjectMapper mapper = new ObjectMapper();
     private static final Logger LOG = LoggerFactory.getLogger(Auth0Users.class);
 
+    private static URI getUrl(String searchQuery, int page, int perPage, boolean includeTotals) {
```

Comment:
Could use javadoc (can be simple though/no need to add params).

Reply:
Updated.